src/seml/commands/start.py

```diff
@@ -25,25 +26,22 @@
     s_if,
 )
 from seml.utils.errors import ArgumentError, ConfigError
+from seml.utils.slurm import (
+    get_cluster_name,
+    get_current_slurm_job_id,
+    get_current_slurm_array_id,
+    get_slurm_jobs,
+)
 
 if TYPE_CHECKING:
-    import pymongo
+    from pymongo.collection import Collection
 
 States = SETTINGS.STATES
 SlurmStates = SETTINGS.SLURM_STATES
 
 
 def get_output_dir_path(config):
-    if 'output_dir' in config['slurm']:
```

Comment:
Shouldn't that be part of the migration?

Reply:
this happens when adding experiments and is very old (0.2 -> 0.3, I think). by now everyone should have adopted the new format.